image.md

```diff
@@ -0,0 +1,140 @@
+# Image
+
+This section defines a format for encoding a container's *filesytem bundle* as a portable *image*.
+An image is a set of files organized in a certain way, and containing all of the necessary data and metadata, for any compliant runtime to verify the validity of the image's contents and to construct a filesystem bundle represented by the image.
+Images are meant to be portable such that they can be used to reliably move filesystem bundles between instances of OCI compliant implementations.
+
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this and other documents of the specification are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
+
+## Image Contents
+
+A compliant image contains the following pieces of information:
+
+1. An OCI [`config.json`](config.md) file.
+This file is REQUIRED to be present in the image.
+The `config.json` file contains the host independent configuration information necessary to execute the container.
+See [`config.json`](config.md) for more information.
+
+1. The root filesystem of the filesystem bundle.
+This directory MUST be present in the image, however it MAY be empty (not contain any files or folders).
```

Comment:
Two reasons:
1 - since the on-the-disk bundle requires it, I thought that for consistency we should require it here.
2 - it seems wrong to have a field point to something that doesn't exist.
If we really want to allow for "no dir at all" then we should let the "rootfs" field to be nil in the config files.